package.json

```diff
@@ -37,21 +37,22 @@
     "airtable": "0.7.2",
     "aws-sdk": "2.534.0",
     "dotenv": "8.0.0",
-    "epsagon": "1.37.0",
+    "epsagon": "^1.106.1",
```

Comment:
Can you remove all `^` ?

By default, we always use fixed version to avoid unforeseen issues.
See https://stackoverflow.com/a/48622855/2391795

Reply:
That makes sense - I pushed a commit to make all version constraints exact.